namespaces/ci_job.yml

```diff
@@ -0,0 +1,48 @@
+
+namespace:
+  name: ci_job
+  type: group
+  description: >
+    Namespace for CI-specific metadata. f.e. Jenkins job run metadata.
+    The namespace contains the details for the metadata of individual job run.
```

Comment:
If this is specific to jenkins, why not use `name: jenkins`, or `name: jenkins_job` if you need to support other jenkins "types"